airbyte-integrations/connectors/source-woocommerce/build.gradle

```diff
@@ -0,0 +1,14 @@
+plugins {
+    id 'airbyte-python'
+    id 'airbyte-docker'
+    id 'airbyte-source-acceptance-test'
+}
+
+airbytePython {
+    moduleDirectory 'source_woocommerce'
+}
+
+dependencies {
+    implementation files(project(':airbyte-integrations:bases:source-acceptance-test').airbyteDocker.outputs)
+    implementation files(project(':airbyte-integrations:bases:base-python').airbyteDocker.outputs)
+}
```

Comment:
Need to `./gradlew format` or just add the new line after in the end of this file.

Reply:
done